VisualizationBase/src/items/ViewItem.cpp

```diff
@@ -204,6 +262,11 @@ void ViewItem::determineChildren()
 				}
 		}
 	}
+	//Must make sure to add the necessary arrow layers here, as sometimes they are added
```

Comment:
Ah sorry, I thought I commented on this, but I think it was when my internet was broken.
Anyway, there are two cases when we add arrows:
1) When working normally with a task view -> then the view is always already added to the scene
2) When loading a task view from memory -> then it is initialized from Json before being added to the scene